tracing-subscriber/src/layer.rs

```diff
@@ -501,6 +501,10 @@ where
         }
     }
 
+    fn current_span(&self) -> span::Current {
```

Comment:
this should probably be `#[inline]`, like the other functions that forward to the inner subscriber?

Reply:
done